virt_lightning/shell.py

```diff
@@ -334,6 +334,32 @@ def go_console(domain):
     ui.Selector(sorted(hv.list_domains()), go_console)
 
 
+def viewer(configuration, name=None, **kwargs):
+    conn = libvirt.open(configuration.libvirt_uri)
+    hv = vl.LibvirtHypervisor(conn)
+
+    def go_viewer(domain):
+        pid = os.fork()
+        if pid == 0:
+            os.close(1)
+            os.close(2)
+            os.execlp(
+                "virt-viewer",
```

Comment:
Could you please add availability of `virt-viewer` binary file in the system?

Reply:
I'm not quite sure what you mean - do you want me to check if it's present and gracefully handle it missing?

Reply:
Yes, I think this is what @zloidemon wants.

Reply:
Cool, I'll figure something out and bump the PR when it's done.

Reply:
Added a little helper to check for 'virt-viewer'. This could potentially be abstracted out a bit, with console (virsh) also using it.